wordcamp.test/provision/vvv-init.sh

```diff
@@ -76,7 +76,11 @@ else
 
 	for i in "${SVN_PLUGINS[@]}"
 	do :
-		svn up $SITE_DIR/wp-content/plugins/$i
+		(
+			cd "${SITE_DIR}/wp-content/plugins/$i" &&
+			svn cleanup &&
+			svn up
+		)
```

Comment:
I don't necessarily have any objection to these, but I'm curious why they're necessary. Are there plugins that have spaces? Why do `cleanup` and `up` need to run within each subfolder?

Reply:
- re: spaces, just good practice, I've been burnt by this a few times in projects
 - `svn cleanup` is because if you cancel mid-provision the svn locks are still in place. I've encountered this in the past when SVN has given up mid-way because of conference wifi, and it's a pain to walk people through if it's new to them

Reply:
Ah, good points :)